### PR TITLE
Fixed #33 -- Added "ticket next steps" suggestions.

### LINKED ISSUE #33: Improve UX of ticket flags with an explanation of next steps
The idea is to add something to [tickethacks.js](https://github.com/django/code.djangoproject.com/blob/master/trac-env/htdocs/tickethacks.js) which translates the "Has patch", "Needs documentation", etc. flags to a prominently displayed paragraph that explains how to move the ticket forward.

For example, "Has patch=Yes" and the rest "no" would say, "To move this ticket forward, a community member needs to review the patch using the [patch review checklist](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/submitting-patches/#patch-review-checklist) and either mark the ticket as "Ready for checkin" if everything looks good, or leave comments for improvement and mark the ticket as "Patch needs improvement".

I'll be happy to provide further guidance on this for anyone interested in tackling the implementation.

### PR DESCRIPTION
**Pull request by timgraham**
screenshot from 2016-10-29 20 24 55

### CODE REVIEW

File: trac-env/htdocs/tickethacks.js [hunk 1]
```diff
@@ -207,4 +207,110 @@ $(function() {
...
+        // case of a regression caused by the ticket
+    } else if (stage == 'Unreviewed') {
+        next_steps.push(
+            "For bugs: reproduce the bug. If it's a regression, " +
+            "<a href='https://docs.djangoproject.com/en/dev/internals/contributing/triaging-tickets/#bisecting-a-regression'>" +
+            "bisect</a> to find the commit where the behavior changed.<br>"
+        );
+        next_steps.push(
+            "For new features or cleanups: give a second opinion of the proposal. " +
+            "Complex proposals may need to be discussed on the " +
```
Comment (MarkusH) [inline trac-env/htdocs/tickethacks.js]: What about DEPs for complex features or changes?

File: trac-env/htdocs/tickethacks.js [hunk 2]
```diff
@@ -207,4 +207,110 @@ $(function() {
...
+    var ticket_status = $('.trac-status').text().trim();
+    var ticket_type = $('.trac-type').text().trim();
+    var has_patch = get_boolean_ticket_flag('has_patch');
+    var patch_needs_improvement = get_boolean_ticket_flag('needs_better_patch');
+    var needs_docs = get_boolean_ticket_flag('needs_docs');
+    var needs_tests = get_boolean_ticket_flag('needs_tests');
+    var next_steps = [];
+    var include_link_to_pr_msg = (
+        "include a link to the pull request in the ticket comment when making " +
+        "that update. The usual format is: [https://github.com/django/django/pull/####&nbsp;PR]."
```
Comment (MarkusH) [inline trac-env/htdocs/tickethacks.js]: Wrap the link syntax in `<code>`?

**Comment by aaugustin** [comment]
Looks good!

File: trac-env/htdocs/tickethacks.js (hunk 1 above)
Comment (timgraham) [inline trac-env/htdocs/tickethacks.js]: I think the DEP requirement isn't common enough to mention it here. Actually the link to django-developers might be too much too. The triager could mention that requirement if they feel it's needed.

**Pull request merged**